src/cr/cube/cube.py

```diff
@@ -275,6 +287,14 @@ def has_means(self):
         """True if cube includes a means measure."""
         return self._measures.means is not None
 
+    @lazyproperty
+    def has_overlaps(self):
+        """True if cube includes a means measure."""
+        return (
+            self._measures.overlaps is not None
+            and self._measures.valid_overlaps is not None
+        )
+
```

Comment:
I'd like to remove this property. This check can be done easly using the `_cube` object in the `cubepart`